x-pack/plugins/cases/public/common/lib/kibana/kibana_react.mock.ts

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
What is the difference between this file and `x-pack/plugins/cases/public/common/lib/kibana/kibana_react.mock.ts` ?